client-go/cmd/client/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"context"
+	"interview-client/internal/consumer"
+	"log"
+
+	"github.com/pkg/errors"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+func main() {
+	ctx := context.Background()
+	serviceAddress := "127.0.0.1:8080"
```

Comment:
One approach here to avoid hard coding the IP is to place the IP in a config file that main() or init() can read in. For example, a staging.json file or a prod.json file.